spec/features/course/assessment/submission/manually_graded_spec.rb

```diff
@@ -214,6 +214,29 @@
         expect(submission.reload.published?).to be(true)
       end
 
+      scenario 'I can grade a submission with delayed_grade_publication' do
```

Comment:
I think there are already feature specs for deployed grade publication, modify that instead ?